openapi.yaml

```diff
@@ -10615,6 +10618,344 @@ components:
+            longitude: -122.425332
+            latitude: 37.758056
+            name: Facebook HQ
+            address: '1 Hacker Way, Menlo Park, CA 94025'
+      description: 'The header of your message can include one type of media content e.g. image, video, document and location.'
+      properties:
+        image:
+          title: Object
+          type: object
+          description: Link to image location
+          properties:
+            link:
+              type: string
+              description: Url string of the image
+              example: 'http://placekitten.com/200/300'
+        video:
+          title: Object
+          type: object
+          description: Link to video location
+          properties:
+            link:
+              type: string
+              description: Url string of the video
+              example: 'https://www.learningcontainer.com/wp-content/uploads/2020/05/sample-mp4-file.mp4'
+        document:
+          title: Object
+          type: object
+          description: Link to document location
+          properties:
+            link:
+              type: string
+              description: Url string of the document
+              example: 'https://s3-eu-west-1.amazonaws.com/hughhopkins.co.uk/sample_media/Sf+flight+itinerary.pdf'
+        location:
+          title: Object
+          type: object
+          description: '“Longitude”, "latitude", "name", and "address"'
+          properties:
+            longitude:
+              type: number
+              description: Longitude value of the location
+              example: -122.425
+            latitude:
+              type: number
+              description: Latitude value of the location
+              example: -122.425  
+            name:
+              type: string
+              description: Name of the location
+              example: 'Facebook HQ'
+            address:
+              type: string
+              description: Address of the location
+              example: '1 Hacker Way, Menlo Park, CA 94025'
+    WAContentButton:
+      title: WAContentButton
+      x-stoplight:
+        id: 8rugv1dziyhzp
+      type: object
+      description: |-
+        The contents of your dynamic button.
+
+        If the button subtype is “url” you will need the following parameters:
+        “Index”, “sub_type”, “text”
+
+        If the button subtype is a “quick reply”, you will need the following parameters:
+        “Index”, “sub_type”, “payload
+      x-examples:
+        Example 1:
+          button:
+            - index: '0'
+              sub_type: quick_reply
+              payload: Yes-Button-Payload
+            - index: '1'
+              sub_type: url
+              payload: 'https://google.com'
+      properties:
+        index:
+          type: string
+          description: The index value of the button this object represents
+          example: '0'
+        sub_type:
+          type: string
+          enum:
+            - quick reply
```

Comment:
hey @reinlabares , there's a small discrepancy between these two lines, one includes an underscrore `"_"` delimiter and the other is space `" "` delimitered. Could you please align for consistency
* https://github.com/whispir/openapi/pull/200/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R10927
* https://github.com/whispir/openapi/pull/200/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R10940

Reply:
Good catch! Thank you. I have addressed this in the latest commit